packages/skuba-dive/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "skuba-dive",
+  "version": "2.0.0",
+  "private": false,
+  "description": "Minimal runtime for skuba",
+  "homepage": "https://github.com/seek-oss/skuba/tree/master/packages/skuba-dive#readme",
+  "bugs": {
+    "url": "https://github.com/seek-oss/skuba/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/seek-oss/skuba.git"
+  },
+  "license": "MIT",
+  "sideEffects": true,
+  "main": "./lib/index.js",
+  "module": "./lib/index.js",
+  "types": "./lib/index.d.ts",
+  "files": [
+    "lib/**/*.d.ts",
+    "lib/**/*.js",
+    "lib/**/*.js.map",
+    "register.js"
+  ],
+  "scripts": {
+    "build": "pnpm skuba build",
+    "format": "pnpm skuba format",
+    "lint": "pnpm skuba lint",
+    "prepack": "pnpm skuba build",
```

Comment:
https://www.npmjs.com/package/skuba-dive/v/2.0.1-dive-migration-20240325115531?activeTab=code

Seems to work, unless the pre-publish script is the one running..?

Reply:
Confirmed this is required to populate a build artefacts for the repo